scss/application.scss

```diff
@@ -1 +1,4 @@
 @import "equalizer";
+@import "colors";
+@import "styles";
+@import url('https://fonts.google.com/specimen/Roboto?selection.family=Roboto');
```

Comment:
there's no need to add the `url(...)` you can do `@import 'https://fonts.google.com/specimen/Roboto?selection.family=Roboto';`

I would also move this before the styles so it loads before them and the font get's defined before it's being used.

I would also change those double quotes for single quotes like the ones I used in the font import in this comment.